src/utilities/texture_atlas.jl

```diff
@@ -500,7 +503,9 @@ end
 rescale_marker(atlas, char, font, markersize) = markersize
 
 function rescale_marker(atlas::TextureAtlas, char::AbstractVector{Char}, font, markersize)
-    return _bcast(markersize) .* marker_scale_factor.(Ref(atlas), char, font)
+    msf = marker_scale_factor(atlas, first(char), font)
+    msfs = fill(msf, length(char))
+    return _bcast(markersize) .* msfs
```

Comment:
These are not going to work with per element markers because they don't have a consistent size in the texture atlas. E.g.:
```julia
julia> Makie.marker_scale_factor.(Ref(Makie.get_texture_atlas()), ['A', 'b', '.', '|', '-'], Ref(Makie.defaultfont()))
5-element Vector{Vec{2, Float32}}:
 [1.0, 1.09375]
 [0.84375, 1.109375]
 [0.46875, 0.46875]
 [0.4375, 1.3125]
 [0.609375, 0.4375]
```

Reply:
Do you have an MWE as well that i can run?